includes/search.inc

```diff
@@ -73,8 +70,7 @@ function bc_views_islandora_solr_query_alter(&$islandora_solr_query) {
  */
 function bc_views_form_islandora_solr_advanced_search_form_alter(&$form, &$form_state) {
 	global $_bc_views_solr_view;
-
-	if (isset($_bc_views_solr_view) || isset($form_state['build_info']['args']['barnard_custom_view'])) {
+	if (isset($form_state['build_info']['args']['barnard_custom_view'])) {

```

Comment:
If i'm following, this condition might not be true when we first use the search form on our content page. Primarily because this form_state variable was triggered after the user preforms their first search. I did this because the AJAX callbacks on the form don't have access to our global variable, but do have access to the form_state.